print/print-apps/oereb/pdfextract.jrxml

```diff
@@ -310,6 +310,8 @@
 	</group>
 	<detail>
 		<part uuid="dbf7b612-de21-4609-8537-6538c611480c">
+			<!-- IMPORTANT NOTE: This report part is only printed if there is a PLR for this real estate to avoid empty pages and tables. -->
+			<printWhenExpression><![CDATA[$P{RealEstate_RestrictionOnLandownershipDataSource} instanceof net.sf.jasperreports.engine.data.JRMapCollectionDataSource]]></printWhenExpression>
```

Comment:
Can we add something like:
`Only if there is at least one PLR in RealEstate_RestrictionOnLandownershipDataSource, it has the type JRMapCollectionDataSource. If it's empty, it is a JREmptyDataSource.`

Otherwise it may be hard to understand, why we check for a type instead of just counting elements.